<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QWidget" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Principal</string>
  </property>
  <widget class="QLabel" name="lbEstado">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>29</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCerrarSesion">
   <property name="geometry">
    <rect>
     <x>293</x>
     <y>192</y>
     <width>88</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar sesi√≥n</string>
   </property>
  </widget>
  <widget class="CameraWidget" name="cameraWidget" native="true">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>49</y>
     <width>192</width>
     <height>137</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QPushButton" name="pbProcesarVideo">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>190</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Procesar video</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CameraWidget</class>
   <extends>QWidget</extends>
   <header>camerawidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
